gui/pause/options/rebind_option.gd

```diff
@@ -24,31 +65,38 @@ func _input(event):
 			update_list()
 
 
-func _process(_delta):
-	update_list()
-
-
 func update_list():
-	key_list.text = join_action_array(InputMap.action_get_events(action_id))
-
-
-func join_action_array(actions) -> String:
 	var output: String = ""
-	for action in actions:
-		if action is InputEventJoypadButton:
-			var buttons = _get_joypad_buttons()
-			if action.button_index > buttons.size():
+	var joy_brand = get_joypad_brand()
+	
+	# Read bindings from this action's input map.
+	var bindings = InputMap.action_get_events(action_id)
+	# List out the action's bindings in a readable format.
+	for bind in bindings:
+		# Add this action's display name to the output.
+		# Joypad inputs are surrounded with (parentheses), others are raw.
+		if bind is InputEventJoypadButton:
+			# Binding is a joypad button.
+			# Print the brand-accurate name for this button, or an error symbol
+			# if the button's not recognized.
+			if bind.button_index > joypad_buttons.size():
 				output += "(?)"
 			else:
-				output += "(%s)" % buttons[action.button_index][get_brand_id()]
-		elif action is InputEventJoypadMotion:
-			output += "(%s)" % get_joypad_motion_name(action.axis, action.axis_value)
+				output += "(%s)" % joypad_buttons[bind.button_index][joy_brand]
+		elif bind is InputEventJoypadMotion:
+			# Binding is some direction on an analog stick.
+			output += "(%s)" % get_joypad_motion_name(bind.axis, bind.axis_value)
 		else:
+			# Binding is a non-joypad action.
+			# Keyboard keys are included here.
 			# TODO: make these translatable
-			output += action.as_text()
+			output += bind.as_text()
+		# Add separator between this and the next binding.
 		output += ", "
+	# Trim off the final separator for visual cleanliness.
 	output = output.trim_suffix(", ")
-	return output
+	
+	key_list.text = output
```

Comment:
The game crashes here for me. "Invalid assignment of property or key 'text' with value of type 'String' on a base object of type 'Nil'."

I believe this happens because `update_list()` is called before `@onready` can trigger, when the translation change notification is received. I'm not sure why this didn't occur while you were working on it, perhaps Godot updates, or perhaps something to do with my own language settings? (Apologies for the belated response to your PR.)